apps/web/src/pages/seats/Error.tsx

```
@@ -1,0 +1,27 @@
+import Button from "@src/components/commons/Button";
+import { Link, useNavigate } from "react-router-dom";
+
+export default function Error() {
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)